aten/src/ATen/native/Distributions.cpp

```diff
@@ -295,4 +295,30 @@ Tensor _s_dirichlet_cpu(const Tensor& alpha, Generator *gen) {
   });
   return ret;
 }
+
+Tensor multinomial_cpu(const Tensor& self, int64_t n_sample, bool with_replacement, Generator *gen) {
+  Tensor result = at::empty({0}, self.options().dtype(kLong));
+  multinomial_out_cpu(result, self, n_sample, with_replacement, gen);
+  return result;
+}
+
+Tensor& multinomial_out_cpu(Tensor& result, const Tensor& self, int64_t n_sample, bool with_replacement, Generator *gen) {
+  TORCH_CHECK(self.dim() >= 1, "multinomial only supports at least 1D tensor, got: ", self.dim(), "D");
```

Comment:
where is this checking coming from?

Reply:
The was this check on CUDA side, not on CPU. I think we don't want to support this.

Reply:
Based on offline conversation, I removed this check. It will be fix in PR.